opencti-platform/opencti-graphql/src/modules/attributes/stixSightingRelationship-registrationAttributes.ts

```diff
@@ -5,12 +5,12 @@ import { STIX_SIGHTING_RELATIONSHIP } from '../../schema/stixSightingRelationshi
 
 export const stixSightingRelationshipsAttributes: { [k: string]: Array<AttributeDefinition> } = {
   [STIX_SIGHTING_RELATIONSHIP]: [
-    { name: 'attribute_count', type: 'numeric', mandatoryType: 'external', editDefault: true, multiple: false, upsert: false, label: 'count' },
-    { name: 'first_seen', type: 'date', mandatoryType: 'customizable', editDefault: true, multiple: false, upsert: false, label: 'first seen' },
-    { name: 'last_seen', type: 'date', mandatoryType: 'customizable', editDefault: true, multiple: false, upsert: false, label: 'last seen' },
-    { name: 'description', type: 'string', mandatoryType: 'customizable', editDefault: true, multiple: false, upsert: true },
-    { name: 'x_opencti_negative', type: 'boolean', mandatoryType: 'customizable', editDefault: true, multiple: false, upsert: false, label: 'False positive' },
-    { name: 'x_opencti_workflow_id', type: 'string', mandatoryType: 'no', editDefault: false, multiple: false, upsert: false },
+    { name: 'attribute_count', label: 'Count', type: 'numeric', precision: 'integer', mandatoryType: 'external', editDefault: true, multiple: false, upsert: false, isFilterable: true },
+    { name: 'first_seen', label: 'First seen', type: 'date', mandatoryType: 'customizable', editDefault: true, multiple: false, upsert: false, isFilterable: true },
+    { name: 'last_seen', label: 'Last seen', type: 'date', mandatoryType: 'customizable', editDefault: true, multiple: false, upsert: false, isFilterable: true },
+    { name: 'description', label: 'Description', type: 'string', format: 'short', mandatoryType: 'customizable', editDefault: true, multiple: false, upsert: true, isFilterable: true },
+    { name: 'x_opencti_negative', label: 'Negative', type: 'boolean', mandatoryType: 'customizable', editDefault: true, multiple: false, upsert: false, isFilterable: true },
```

Comment:
maybe label should be "False Positive" to match before?

Reply:
yes maybe, I'm going to modify the label

Reply:
fixed